content/en/blog/_posts/2022-12-06-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: blog
+title: "Kubernetes v1.26: Electrifying"
+date: 2022-12-6
+slug: kubernetes-v1-26-release
+---
+
+**Authors**: [Kubernetes 1.26 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.26/release-team.md)
+
+It's with immense joy that we announce the release of Kubernetes v1.26!
+
+This release includes a total of 37 enhancements: eleven of them are graduating to Stable, ten are graduating to Beta, and sixteen of them are entering Alpha. We also have two features being deprecated or removed.
```

Comment:
38 enhancements AFAIK.

2 features being deprecated.. AFAIK just one? which one have I missed? https://github.com/kubernetes/enhancements/issues/281

Reply:
Yes, good catch, there was one that was added to the milestone a few days ago, will add it.

There's only one tracked enhancement in the board, but at the time of writing we were also aware of the GlusterFS removal, but the actual total is further down and it's 12, with API changes. We can either just mention the 1 tracked removal, or the total identified - either way I need to update this.

Reply:
They aren't just smaller API changes though: we have the klog removal, GlusterFS removal, in-tree credential code, etc - they are all listed in the "Deprecations and Removals" section further down this article. There are 3 API removals (one of them with significant impact), 3 deprecations, the rest of the removals are not substantially different from one another in terms of relevance, which makes it difficult to separate. 

In terms of tracking, only the Dynamic Kubelet Configuration exists as a deprecation. In reality, we have (at least) the 12 identified ones.

One way around this is to mention the  3 "major" ones, which are exactly the ones that are described as a Major theme (CRI, GlusterFS, Cinder), and say something like:

> [...] We also have 12 deprecations and removals, 3 of which detailed below due to their impact.